scripts/participateInAuction.js

```diff
@@ -7,8 +11,16 @@ const { Gastimator } = require("./gasStation")
 const gasPriceUrl = require("../gas-config")
 const GasStation = new Gastimator()
 
+// const STATE_POOLING = 0 // Pooling
+const STATE_POOLING_ENDED = 1 // PoolingEnded
+const STATE_UNLOCKING_MGN = 2 // DepositWithdrawnFromDx
+// const STATE_MGN_UNLOCKED = 3 // MgnUnlocked
+
 async function participateInAuction(coordinatorAddress, network) {
-  console.log("[%s] Do articipate in auction", coordinatorAddress)
+  const debugAuction = Debug("DEBUG-participate:" + coordinatorAddress)
```

Comment:
Suggest including space after ":"

Reply:
I didn't add a space cause that's the name of the logger.
We use the `:` to create kind of namespaces for the logger, so we can control the debug level with a finer grain.
example of a logger could be `DEBUG-services:service-a:repository-a`

In this case, I created a logger per pool, cause the bots now support any number of pools. So you can increase the log level for only one:

![image](https://user-images.githubusercontent.com/2352112/57841621-30a99f80-77cb-11e9-85ef-2aed026a6f6e.png)

Reply:
I'd call this debugWithCoordinatorAddress or something like that?